Rodgers-M/Bright_events
# Fix email validation and implement events searching.

**What does this PR do?**

implements events searching and email validation

**Description of tasks to be completed?**

Validate email during registration. users should have unique emails.

**How should this be tested manually?**

Create a few events in the app and search for events based on the two parameters.

Users should be able to search events based on category or location.

Replace the search field on landing page with a get started button.
